lib/Scraped/json.rb

```diff
@@ -0,0 +1,18 @@
+require 'scraped'
+
+class Scraped
```

Comment:
This is only working because you're running off an old version of `scraped`. `Scraped` is module, not a class.